### ISSUE TITLE
make strings translatable for i18n usage

### ISSUE THREAD

**Issue by patroqueeet** [Execut3/django-discount#2]
just seen that by reading the code...

**Comment by Execut3** [comment]
Yes, right now is in Persian, will update i18n support.

**Comment by patroqueeet** [comment]
I have some usage for Switzerland in mind... your base code looks quite solid and there is no comparable lib out there... so maybe we want to raise it to a more professional level (at least when its listed on pypi).

**Comment by Execut3** [comment]
yes will appreciate any comment or idea to make it more complete. feel free for any suggestions ;)
